sdk/cognitivelanguage/Azure.AI.Language.Conversations/src/Models/LuisTargetIntentResult.cs

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.AI.Language.Conversations.Models
+{
+    /// <summary> It is a wrap up of LUIS Generally Available response. </summary>
+    [CodeGenModel("LuisTargetIntentResult")]
+    public partial class LuisTargetIntentResult : TargetIntentResult
+    {
+        /// <summary> The actual response from a LUIS Generally Available application. </summary>
+        [CodeGenMember("Result")]
+        internal object InternalResult { get; }
+
+        /// <summary> Binary Data wrapper for the response from a LUIS Generally Available application. </summary>
+        public BinaryData Result { get => BinaryData.FromObjectAsJson(InternalResult); }
+    }
```

Comment:
Cache this instead e.g.

```c#
private object _internalResult;

[CodeGenMember("Result")]
private object InternalResult { get; }

public BinaryData Result => _internalResult ??= BinaryData.FromObjectAsJson(InternalResult);
```

Tip: Using just `=> ...` for a property is the same (but less typing) than `{ get => ... }`.